packages/account/src/Components/sent-email-modal/__tests__/sent-email-modal.spec.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import SendEmailModal from '../sent-email-modal';
+
+let onClose, onClickSendEmail;
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+    isDesktop: jest.fn(),
+}));
+
+describe('<SendEmailModal/>', () => {
+    beforeAll(() => {
+        const modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    beforeEach(() => {
+        onClose = jest.fn();
+        onClickSendEmail = jest.fn();
+    });
```

Comment:
should we do this assignment before every test? maybe we can just make it one time in the beginning of describing our cases?